pkg/controllers/awsloadbalancercontroller/rbac_clusterrole.go

```diff
@@ -0,0 +1,131 @@
+package awsloadbalancercontroller
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	albo "github.com/openshift/aws-load-balancer-operator/api/v1alpha1"
+)
+
+// ensureClusterRoles creates and maintains clusterroles required by the aws-load-balancer controller.
+func (r *AWSLoadBalancerControllerReconciler) ensureClusterRoles(ctx context.Context, controller *albo.AWSLoadBalancerController) error {
```

Comment:
Let's change this and the corresponding functions to behave like there is a single `ClusterRole`.

Reply:
Addressed